File: Ruby/fizzbuzz.rb
def fizzbuzz(from, to)
    (from..to).each { |i|
        case 0
        when i % 15
            puts "FizzBuzz"
        when i % 5
            puts "Buzz"
        when i % 3
            puts "Fizz"
        when 0
            puts i
        end
    }
end

fizzbuzz 1, 100

File: Scala/FizzBuzz.scala
object FizzBuzz {
  def main(args: Array[String]) {
    (1 to 100).map( x => {
      if (x % 15 == 0) {
        println("FizzBuzz")
      } else if (x % 5 == 0) {
        println("Buzz")
      } else if (x % 3 == 0) {
        println("Fizz")
      } else {
        println(x)
      }
    })
  }
}
